app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss

```diff
@@ -0,0 +1,113 @@
+.tabs-container {
+}
+
+.tabs {
+  display: flex;
+  background-color: $COLOR--white;
+  border-top: 1px solid $COLOR--gray-91;
+
+  @media (max-width: $SCREEN_XS_MAX) {
+    flex-direction: column;
+  }
+}
+
+.tab {
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding: 0 10px;
+  cursor: pointer;
+  color: $COLOR--gray-47;
+  fill: $COLOR--gray-47;
+  transition: fill 0.2s, color 0.2s;
+  height: 40px;
+  box-sizing: border-box;
+  background-color: transparent;
+  border: 0;
+  outline: 0;
+
+  &:hover {
+    color: #000;
+    fill: #000;
+  }
+
+  &:after {
+    content: '';
+    width: 1px;
+    height: 20px;
+    position: absolute;
+    right: 0;
+    background: $COLOR--gray-80;
+    top: 10px;
+
+    @media (max-width: $SCREEN_XS_MAX) {
+      display: none;
+    }
+  }
+
+  &:focus,
+  &.active,
+  &.active:hover {
+    color: $COLOR--topaz;
+    fill: $COLOR--topaz;
+  }
+
+  @media (max-width: $SCREEN_XS_MAX) {
+    border-bottom: 1px solid $COLOR--gray-80;
+  }
+}
+
+.tab-icon {
+  display: inline-block;
+  margin-top: 4px;
+
+  svg {
+    width: 20px;
+    height: 20px;
+  }
+}
+
+.tab-label {
+  font-size: 13px;
+  text-transform: uppercase;
+  font-family: $FONT-SEMIBOLD;
+  font-weight: bold;
```

Comment:
Please do not use font-weight property. We have font types for all required weights.